index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <head><meta charset="utf-8">
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.0.1/styles/base16/solarized-light.min.css"><link rel="stylesheet" href="./spec.css">
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.0.1/styles/base16/solarized-light.min.css" integrity="sha512-1l+yD1g5lzzMJrdmD8L5N6FblzkAnd9dzjw03oYJkhe8NpmyzkqahigJDRhs1SuxYvZAsHIY50pqa/rzrWzArA==" crossorigin="anonymous"><link rel="stylesheet" href="./spec.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/github.min.css" integrity="sha512-iDO4iL6PUvPt54yn8f29JSkoMrScHRIMArsHJtOTGg2yTHWT7dfvziWeXIneGy+wGwbrhGMg1a1ntA+sNA+atg==" crossorigin="anonymous">
```

Comment:
why would we need to add this?

Reply:
Well, this ensures that the webpage author receives the file that they expect to receive from the CDNs. CDNs can get attacked and start serving malicious files to everyone visiting your page.

Reply:
Sure, but this is a spec proposal. Why is that important?

Reply:
I don't think it matters what the particular webpage is about, it's more to do w/ being sure that you are getting what you expect to be getting.

Regarding the `crossorigin="anonymous"`, maybe read about on mdn, but i haven't had to think about cors in too long anymore.

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin

Reply:
Right, but i mean, there's nothing to compromise here - this would just be security for its own sake, but there's no damage to prevent.

Reply:
This is the tool that autogenerates that whole replacement of the tag: https://www.srihash.org/